runtime/thunk/thunk.ts

```diff
@@ -166,12 +166,13 @@ async function doFindAndExecutePackFunction<T extends FormulaSpecification>({
             return selectedAuthentication.getConnectionUserId.execute(params as [string, string], executionContext);
           }
           break;
-        case MetadataFormulaType.ParameterAutocomplete:
-          const parentFormula = findParentFormula(manifest, formulaSpec);
```

Comment:
Oh geez, was this mis-named before?

Reply:
Previously findParentFormula was only used for ParameterAutocomplete. For ValidateParameters, we use much of the same logic so this was a side effect of extracting out findParentFormula to be used in both helpers.